src/scraper.ts

```diff
@@ -0,0 +1,57 @@
+import { load, type CheerioAPI } from "cheerio";
+import decodeEntities from "html-entities-decode";
+
+export type CheerioNode = {
+	name: string;
+	children: CheerioNode[];
+	attribs: { [attr: string]: string };
+};
+
+if (process.env.SCRAPE_CACHE) {
+	if (!Number.isInteger(+process.env.SCRAPE_CACHE))
+		throw new TypeError(
+			"Environment variable SCRAPE_CACHE should be an integer",
+		);
+	if (+process.env.SCRAPE_CACHE < 1)
```

Comment:
Zero should be accepted to disable the cache. Negative values could also be accepted for caching indefinitely, but that isn't really necessary.